### ISSUE TITLE
A few questions...

### ISSUE THREAD

**Issue by asigalov61** [salu133445/mmt#1]
Hey Hao-Wen,

I have a few questions about this implementation and results if you do not mind...

1) What were the hyperparameters for your best model (MTMT-APE)?
2) Why your best loss is so high (1.6)?
3) Did you train on the full LMD? (I think you said yes but I am not sure from the repo).
4) Can you share pre-trained models?

Also, I wanted to suggest using better MIDI rendering. Your samples are hard to listen to. I can suggest great SF2s, if you are interested.

Let me know.

Alex
